### DISCUSSION TITLE
Plugin support?

### DISCUSSION THREAD

**Discussion by papay88** [LTmadness/ValheimBetterServerConfig#32] [Q&A] [answered]
Hello, have you stopped supporting this plugin? There have been no updates for a long time, and the plugin page on the nexusmods website has disappeared.

**Comment by LTmadness** [comment, marked as answer]
Hi, the mod still supported, I only stoped uploading any of my mods on
nexus, I'm working on few new features, but for now this mod is working
fine on current version of Valheim

**Reply by papay88** [reply]
It's great to hear that you are continuing to work on the mod. In my opinion, this is a very important mod for game servers.
Can you find out why you left nexus?
With love from Russia ;)

**Comment by LTmadness** [comment]
Because Nexus mods allows miss-leading people and admins are allowed to
leak private messages
